packages/web3-shared/evm/src/helpers/isValidDomain.ts

```diff
@@ -1,4 +1,4 @@
-import punycode from 'punycode/'
+import punycode from 'punycode'
```

Comment:
this is a trick to import the npm version of the module instead of the built-in NodeJS module